clients/bigquery/dialect/dialect_test.go

```diff
@@ -62,69 +84,85 @@ func TestBigQueryDialect_KindForDataType(t *testing.T) {
 			assert.Equal(t, int32(0), kd.ExtendedDecimalDetails.Scale())
 			assert.Equal(t, "NUMERIC(5, 0)", kd.ExtendedDecimalDetails.BigQueryKind())
 		}
+		{
+			// Numeric(5, 2)
+			kd, err := dialect.KindForDataType("numeric(5, 2)", "")
+			assert.NoError(t, err)
+			assert.Equal(t, typing.EDecimal.Kind, kd.Kind)
+			assert.Equal(t, int32(5), kd.ExtendedDecimalDetails.Precision())
+			assert.Equal(t, int32(2), kd.ExtendedDecimalDetails.Scale())
+		}
+		{
+			// BigNumeric(5, 2)
+			kd, err := dialect.KindForDataType("bignumeric(5, 2)", "")
+			assert.NoError(t, err)
+			assert.Equal(t, typing.EDecimal.Kind, kd.Kind)
+			assert.Equal(t, int32(5), kd.ExtendedDecimalDetails.Precision())
+			assert.Equal(t, int32(2), kd.ExtendedDecimalDetails.Scale())
+		}
 	}
-
-	bqColToExpectedKind := map[string]typing.KindDetails{
-		// Number
-		"numeric":           typing.EDecimal,
-		"numeric(5, 2)":     typing.EDecimal,
-		"numeric(8, 6)":     typing.EDecimal,
-		"bignumeric(38, 2)": typing.EDecimal,
-
-		// Integer
-		"int":     typing.Integer,
-		"integer": typing.Integer,
-		"inT64":   typing.Integer,
-		// String
-		"varchar":     typing.String,
-		"string":      typing.String,
-		"sTriNG":      typing.String,
-		"STRING (10)": typing.String,
-		// Array
-		"array<integer>": typing.Array,
-		"array<string>":  typing.Array,
-		// Boolean
-		"bool":    typing.Boolean,
-		"boolean": typing.Boolean,
-		// Struct
-		"STRUCT<foo STRING>": typing.Struct,
-		"record":             typing.Struct,
-		"json":               typing.Struct,
-		// Datetime
-		"datetime":  typing.TimestampNTZ,
-		"timestamp": typing.TimestampTZ,
-		"time":      typing.Time,
-		"date":      typing.Date,
-		//Invalid
-		"foo":    typing.Invalid,
-		"foofoo": typing.Invalid,
-		"":       typing.Invalid,
+	{
+		// Integers
+		for _, intKind := range []string{"int", "integer", "int64"} {
```

Comment:
Before we were testing `"inT64"` (with a capital T) - want to change it back to that so we're still checking it's not case-sensitive?